src/integration-tests/socks-echo/base-spec.core-env.ts

```diff
@@ -332,10 +332,7 @@ export function socksEchoTestDescription(useChurn:boolean) {
     }).then(done);
   });
 
-  // Disabled because HOST_UNREACHABLE is not yet exposed in freedom-for-chrome's
-  // implementation of the core.tcpsocket API.
-  //  https://github.com/freedomjs/freedom-for-chrome/issues/73
-  xit('attempt to connect to a nonexistent DNS name', (done) => {
+  it('attempt to connect to a nonexistent DNS name', (done) => {
```

Comment:
I wouldn't expect it to help.  That sounds like something that could be caused by a missing error check in freedom-for-chrome's core.tcpsocket.